src/content/structured/components/toggle-buttons/accessibility.mdx

```diff
@@ -29,7 +29,7 @@ The ICDS toggle button is based upon the [ICDS button](../components/button) so
 
 A toggle button group can be set to either single-select or multi-select, and has settings that allow automatic selection or manual selection of its options as each toggle button is navigated through. This helps to speed up selection in a single-select toggle button group. However, this does change the keyboard interaction.
 
-For a single select group, the whole group is focussed upon using the ‘tab’ key and the options are navigated using the ‘arrow’ keys, such as with a [radio button group](../components/radio-buttons). For a multi-select group each toggle button in the group is individually focusable with the ‘tab’ key.
+For a single select group, the whole group is focussed upon using the ‘tab’ key and the options are navigated using the ‘arrow’ keys, such as with a [radio button group](../components/radios). For a multi-select group each toggle button in the group is individually focusable with the ‘tab’ key.
```

Comment:
I think this should just be radio as well, there doesn't seem to be a radios link either